mintpy/save_roipac.py

```diff
@@ -46,13 +46,16 @@ def read_data(inps):
     k = atr['FILE_TYPE']
     if k == 'velocity':
         # read/prepare data
-        data = readfile.read(inps.file)[0]
+        if inps.dset is None:
+            data = readfile.read(inps.file)[0]
+        else:
+            data = readfile.read(inps.file, datasetName = inps.dset)[0]
 
         # velocity to displacement
-        print('convert velocity to displacement for {}'.format(atr['DATE12']))
-        date1, date2 = atr['DATE12'].split('_')
-        dt1, dt2 = ptime.date_list2vector([date1, date2])[0]
-        data *= (dt2 - dt1).days / 365.25
+        #print('convert velocity to displacement for {}'.format(atr['DATE12']))
+        #date1, date2 = atr['DATE12'].split('_')
+        #dt1, dt2 = ptime.date_list2vector([date1, date2])[0]
+        #data *= (dt2 - dt1).days / 365.25
 
```

Comment:
What's the rationale to comment out the conversion part? Quite some modeling codes expect the old ROI_PAC file, which does not have a velocity format. Thus, converting it into displacement is needed.

If your goal is saving the other datasets inside the velocity file, such as the step function, seasonal fitting etc., how about adding an if check here, to keep the old behavior for `velocity` dataset only?

Reply:
Hi @yunjunz, sorry for not providing the rationale for commenting out the lines of code. Yeah, you're right I was thinking that for the specific velocity dataset such as step, functional, and seasonal fitting to not convert it to displacement. Yeah, I forgot to consider the velocity dataset for this will need conversion to displacement. 

I will edit this PR based on your suggestion.